.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname $0)/_/husky.sh"
 
-yarn format && yarn lint
```

Comment:
This is a breaking change, as previously yarn format failing would cause pre-commit to fail, and it does not anymore. Probably want `set -x` on verification scripts like this.

Reply:
That's a good catch. I tested `yarn format && yarn lint` on Windows PowerShell just now and it seems to be working now while it wasn't earlier. I've added it #163, will need to drop the old commit from this pull request once #161 and #163 are merged.